modules/docs/arrow-docs/docs/docs/arrow-core/typeclasses/alternative/README.md

```diff
@@ -63,43 +63,33 @@ Option.monadCombine().run {
 
 #### Kind<F, A>.some
 
-`fun <A> Kind<F, A>.some(): Kind<F, SequenceK<A>>`
+`fun <A> Kind<F, A>.some(): Kind<F, NonEmptyList<A>>`
 
-Repeats the current computation, lazily collecting its results into a sequence, until it fails. It is required that the computation succeeds at least once.
+Repeats the current computation, collecting its results into a non-empty list, until it fails. This will loop forever if the computation cannot or does not fail.
 
-```kotlin:ank
+```kotlin
```

Comment:
This and the many example cannot be run with Option because Option is pure and thus never fails when retried. Something like OptionT<ForIO> or similar is better suited here. (Best would be a parser example)